src/tree/ContainerAppTreeItem.ts

```diff
@@ -143,18 +148,28 @@ export class ContainerAppTreeItem extends AzExtParentTreeItem implements IAzureR
         }
 
         const concreteContainerAppEnvelope = <Concrete<ContainerApp>>containerAppEnvelope;
-        const options: AzExtRequestPrepareOptions = {
-            method: 'POST',
-            queryParameters: { 'api-version': '2021-03-01' },
-            pathTemplate: `${this.id}/listSecrets`,
-        };
-
-        const response = await sendRequestWithTimeout(context, options, 5000, this.subscription.credentials);
-        // if 204, needs to be an empty []
-        concreteContainerAppEnvelope.configuration.secrets = response.status === 204 ? [] : <Secret[]>response.parsedBody;
+
+        // bug that returns 204 if there are no secrets
```

Comment:
Has a bug been filed on the SDK for this?

Reply:
It does now 😐